metadata.yaml

```diff
@@ -1,4 +1,9 @@
 missions:
+- id: cache
+  name: Cache
+  description: The Caching mission demonstrates how to use a cache to improve the response time of applications. 
```

Comment:
So is it "Cache" or "Caching"? :-)

Reply:
Agree, I think it should be `The Cache mission ...`

Reply:
And here https://github.com/fabric8-launcher/launcher-documentation/wiki, we call it _Cache Service_. We're the kings of consistency.

Reply:
:-)

I prefer Cache.